modules/mono/glue/GodotSharp/GodotSharp/Core/Callable.cs

```diff
@@ -72,7 +72,7 @@ public Callable(Object target, StringName method)
         /// <param name="delegate">Delegate method that will be called.</param>
         public Callable(Delegate @delegate)
         {
-            _target = null;
+            _target = @delegate.Target as Object;
```

Comment:
The passed in `@delegate` can be `null`.
```suggestion
            _target = @delegate?.Target as Object;
```